automation/test.sh

```diff
@@ -92,7 +92,9 @@ if [ ! -d "cluster-up/cluster/$KUBEVIRT_PROVIDER" ]; then
 fi
 
 if [[ $TARGET =~ sriov.* ]]; then
-  export KUBEVIRT_NUM_NODES=3
+  if [[ $TARGET =~ kind.* ]]; then
+    export KUBEVIRT_NUM_NODES=3
+  fi
```

Comment:
If the number of nodes is not specified here for k3d, where is it specified?

Reply:
hardcoded in the provider, see PR desc please

Reply:
Thank you for the explanation.
Could you please add it to the commit message?